src/background/providers/cloudflare.test.ts

```diff
@@ -355,8 +360,10 @@ describe('redemption', () => {
             expect(result).toEqual({ redirectUrl: details.url });
             // Expect redeemInfo to be set.
             const redeemInfo = provider['redeemInfo'];
-            expect(redeemInfo!.requestId).toEqual(details.requestId);
-            expect(redeemInfo!.token.toString()).toEqual(tokens[0].toString());
+            if (redeemInfo) {
```

Comment:
Could this be more explicit whether it's checking for `false`, `null`, or `undefined`?